tests/integration/pob_suite.go

```diff
@@ -289,6 +296,363 @@ func (s *POBIntegrationTestSuite) TestValidBids() {
+
+		// check bid2 was included first
+		VerifyBlock(s.T(), block, 0, TxHash(txs[1]), bundledTxs2)
+
+		// check escrow balance
+		escrowAcctBalanceAfterBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+		expectedIncrement := escrowAddressIncrement(bidAmt.Add(params.MinBidIncrement.Add(bidAmt)).Amount, params.ProposerFee)
+		require.Equal(s.T(), escrowAcctBalanceBeforeBid+expectedIncrement, escrowAcctBalanceAfterBid)
+
+		// check next block
+		WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height+2)
+		block = Block(s.T(), s.chain.(*cosmos.CosmosChain), int64(height+2))
+
+		// check bid1 was included second
+		VerifyBlock(s.T(), block, 0, TxHash(txs[0]), bundledTxs)
+	})
+
+	s.Run("Multiple bid transactions with second bid being smaller than min bid increment", func() {
+		// escrow account balance
+		escrowAcctBalanceBeforeBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+
+		// create bid 1
+		// bank-send msg
+		msg := Tx{
+			User:              s.user1,
+			Msgs:              []sdk.Msg{banktypes.NewMsgSend(s.user1.Address(), s.user2.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))},
+			SequenceIncrement: 1,
+		}
+		// create bid1
+		bidAmt := params.ReserveFee
+		bid1, bundledTxs := CreateAuctionBidMsg(s.T(), context.Background(), s.user1, s.chain.(*cosmos.CosmosChain), bidAmt, []Tx{msg})
+
+		// create bid 2
+		msg2 := Tx{
+			User:              s.user2,
+			Msgs:              []sdk.Msg{banktypes.NewMsgSend(s.user2.Address(), s.user3.Address(), sdk.NewCoins(sdk.NewCoin("stake", sdk.NewInt(100))))},
+			SequenceIncrement: 1,
+		}
+		// create bid2 w/ higher bid than bid1
+		bid2, _ := CreateAuctionBidMsg(s.T(), context.Background(), s.user2, s.chain.(*cosmos.CosmosChain), bidAmt, []Tx{msg2})
+
+		// get chain height
+		height, err := s.chain.(*cosmos.CosmosChain).Height(context.Background())
+		require.NoError(s.T(), err)
+
+		// broadcast both bids
+		txs := BroadcastTxs(s.T(), context.Background(), s.chain.(*cosmos.CosmosChain), []Tx{
+			{
+				User:   s.user1,
+				Msgs:   []sdk.Msg{bid1},
+				Height: height + 1,
+			},
+			{
+				User:       s.user2,
+				Msgs:       []sdk.Msg{bid2},
+				Height:     height + 1,
+				ExpectFail: true,
+			},
+		})
+
+		// query next block
+		WaitForHeight(s.T(), s.chain.(*cosmos.CosmosChain), height+1)
+		block := Block(s.T(), s.chain.(*cosmos.CosmosChain), int64(height+1))
+
+		// check bid2 was included first
+		VerifyBlock(s.T(), block, 0, TxHash(txs[0]), bundledTxs)
+
+		// check escrow balance
+		escrowAcctBalanceAfterBid := QueryAccountBalance(s.T(), s.chain, escrowAddr, params.ReserveFee.Denom)
+		expectedIncrement := escrowAddressIncrement(bidAmt.Amount, params.ProposerFee)
+		require.Equal(s.T(), escrowAcctBalanceBeforeBid+expectedIncrement, escrowAcctBalanceAfterBid)
+	})
+
+	s.Run("Multiple bid transactions from diff accounts with second bid being smaller than min bid increment", func() {
```

Comment:
this test case is essentially the same as the last no?

Reply:
Yea, I think so as well the difference is that bids are signed by diff accounts